providers/loopia/types.go

```diff
@@ -51,13 +51,6 @@ type structMemberInt struct {
 
 func (m structMemberInt) structMember() {}
 
-type structMemberBool struct {
```

Comment:
These might be generated from another source.  Leave in.

Reply:
Do you mean commenting this out instead of removing?
I'm afraid leaving without changes will produce staticcheck warnings (that may be misleading a bit)

Reply:
oh, duh!

hmm.... yes, I guess comment them out.

Reply:
Commented out